grafast/grafserv/package.json

```diff
@@ -70,10 +70,12 @@
   "homepage": "https://github.com/graphile/crystal/tree/main/grafast/grafserv",
   "dependencies": {
     "@graphile/lru": "workspace:^",
+    "aws-lambda": "^1.0.7",
     "debug": "^4.3.4",
     "eventemitter3": "^5.0.1",
     "graphile-config": "workspace:^",
     "graphql-ws": "^5.14.0",
+    "modern-async": "^1.1.4",
```

Comment:
`aws-lambda` and `modern-async` should **not** be dependencies - only the lambda adaptor needs them, we shouldn't install them for everyone else. Please move them to `peerDependencies` and make them optional via `peerDependenciesMeta`.

Reply:
I am not a huge fan of the idea of copy-pasting type declarations from a 3rd party package, since a peer dep will accomplish all the same things wrt isolating graphile from dep sprawl, while making it easier to track updates in the 3rd party package. So if it's all the same to you, I'll make it a peer dep and keep the import.

Reply:
I meant that you already have `@types/aws-lambda` so you shouldn't _also_ need `aws-lambda` since we're only using the types?